privacy.html

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+---
+
+<div class="row justify-content-center">
+  <div class="col-10">
+    <h1>Privacy Policy for Two Factor Auth</h1>
+    <p>At Two Factor Auth, accessible from https://twofactorauth.org/, one of our main priorities is the privacy of our
+      visitors. This Privacy Policy document contains types of information that is collected and recorded by Two Factor
+      Auth and how we use it.</p>
+    <h2>General Data Protection Regulation (GDPR)</h2>
+    <p>We are a Data Controller of your information.</p>
+    <p>Two Factor Auth legal basis for collecting and using the personal information described in this Privacy Policy
+      depends on the Personal Information we collect and the specific context in which we collect the information:</p>
+    <ul>
+      <li>Two Factor Auth needs to perform a contract with you</li>
```

Comment:
?

Reply:
This was just from a privacy policy template I found.